src/api/app/jobs/full_text_index_job.rb

```diff
@@ -0,0 +1,23 @@
+class FullTextIndexJob < ApplicationJob
+  queue_as :quick
+
+  def perform
+    return if Rails.env.test? || Rails.env.development?
```

Comment:
I prefer to check if is not `Rails.env.production?`, some can have other environments...

Reply:
> I prefer to check if is not Rails.env.production?, some can have other environments...

Done.